tasks/certificate.yml

```
@@ -0,0 +1,20 @@
+---
+
+- name: Ensure target directory exists
+  file:
+    dest: "{{ bbb_ssl_cert | dirname }}"
+    state: directory
+    mode: '0755'
+
+- import_tasks: certificate-letsencrypt.yml
+  when: bbb_letsencrypt_enable|bool
+
+- import_tasks: certificate-custom.yml
+  when: not bbb_letsencrypt_enable|bool
+
+- name: Force private key file permissions
+  file:
+    path: "{{ bbb_ssl_key }}"
+    owner: "root"
+    group: "{{'ssl-cert' if ansible_os_family == 'Debian' else 'root' }}"
+    mode: '0440'
```

```suggestion
    mode: '0440'

```
Add newline in end of file